src/modal/modal.js

```diff
@@ -291,6 +343,13 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.stackedMap'])
           openedClasses.remove(modalBodyClass, modalInstance);
           appendToElement.toggleClass(modalBodyClass, openedClasses.hasKey(modalBodyClass));
           toggleTopWindowClass(true);
+
+          // reset any right padding set to body when opening the modal
+          // make sure no other modal is open before resetting
+          if(modalWindow.appendTo === 'body' && openedWindows.length() === 0){
+            var body = $document.find('body');
+            body.css('padding-right', modalWindow.modalDomEl.originalBodyRightPadding);
```

Comment:
This seems way too opinionated - this will make it near impossible for a user to override this, and is not a good solution.